moveit_ros/moveit_servo/src/utils/command.cpp

```diff
@@ -94,24 +94,31 @@ JointDeltaResult jointDeltaFromJointJog(const JointJogCommand& command, const mo
 
   velocities.setZero();
   bool names_valid = true;
-
-  for (size_t i = 0; i < command.names.size(); ++i)
+  bool command_valid = true;
```

Comment:
Do we need both `command_valid` and `names_valid` as separate variables? Or can they both set the same variable to false?